docs/content/releases.md

```diff
@@ -6,6 +6,50 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+## NGINX Ingress Controller 2.4.0
+
+04 Oct 2022
+
+FEATURES:
+
+* [2986](https://github.com/nginxinc/kubernetes-ingress/pull/2986) Batch reloads at runtime.
+* [2914](https://github.com/nginxinc/kubernetes-ingress/pull/2914) Support watching multiple namespaces.
+* [2884](https://github.com/nginxinc/kubernetes-ingress/pull/2884) Include year in logs.
+* [2993](https://github.com/nginxinc/kubernetes-ingress/pull/2993) Accept proxy protocol when TLS passthrough enabled.
+* [3041](https://github.com/nginxinc/kubernetes-ingress/pull/3041) Support external name service for TansportServer.
+* [3087](https://github.com/nginxinc/kubernetes-ingress/pull/3087) Allow omitting the default server secret from Helm installs.
+* [2939](https://github.com/nginxinc/kubernetes-ingress/pull/2939) Add support for wildcard hostname in VirutalServer.
+
+IMPROVEMENTS:
+
+* [2912](https://github.com/nginxinc/kubernetes-ingress/pull/2912) NIC + NAP DoS images.
+* [3040](https://github.com/nginxinc/kubernetes-ingress/pull/3040) Add command line argument to manually disable IPV6 listeners for unsupported clusters.
+* [3088](https://github.com/nginxinc/kubernetes-ingress/pull/3088) Feat/ignore helm secrets.
```

Comment:
I know this is lifted from the commit message, but I'd probably reword in for here - maybe "Improve secret filtering" or something.